health-monitors/syslog-health-monitor/pkg/common/common.go

```diff
@@ -28,6 +29,13 @@ import (
 	"github.com/thedatashed/xlsxreader"
 )
 
+var (
+	// XIDPattern is a simple pattern to detect the presence of "Xid" in log messages.
+	// This is useful for filtering out XID-related messages that should be handled
+	// by the XID handler instead of other error handlers.
+	XIDPattern = regexp.MustCompile(`\bXid\b`)
```

Comment:
Can we use the pattern we have at health-monitors/syslog-health-monitor/pkg/xid/parser/csv.go?